assets/process-html.js

```diff
@@ -0,0 +1,22 @@
+var { exec } = require('child_process');
+var path = require("path");
+
+var glob = require("glob");
+var env = process.argv.slice(2)[0];
+
+
+glob("./assets/templates/*.html", '', function (er, files) {
+    files.forEach(function(file){
+    exec('npm run html-processor ' + file + ' -- --output ./'+env+'/'+path.basename(file)+' -e '+env, (err, stdout, stderr) => {
```

Comment:
Is there any chance of using template literals here?

Reply:
Not sure but I'll try!